assessment.rb

```diff
@@ -2,11 +2,40 @@
 # Assessment answers to:
 
 # question 2:
-
+licensee = licensee.create( surname: 'Bishop', given_name: 'Caren', height: 63, weight: 132, dob: '1943-09-26', gender: 'f')
```

Comment:
`licensee = Licensee.create ...`

`Licensee` is the name of the class, and that's where the `.create` method lives.